Client/Frontend/Browser/Tab.swift

```diff
@@ -399,7 +404,16 @@ class Tab: NSObject {
     }
 
     func reload() {
-        let userAgent: String? = desktopSite ? UserAgent.desktopUserAgent() : nil
+        
+        var userAgent: String?
```

Comment:
Rather than specifying the `userAgent` here, we should be specifying the `mobileUA`:

```
var mobileUA: String? = nil

if #available(iOS 13.0, *) {
    mobileUA = UserAgent.mobileUserAgent()
}

let userAgent: String? = desktopSite ? UserAgent.desktopUserAgent() : mobileUA
```

This is _much_ cleaner IMO.

@danishjafri88 when seeing code duplication like this (`userAgent = desktopSite ? UserAgent.desktopUserAgent() : `), it is best to try to identify how we can abstract the duplicated portions.